Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ISISIndirectDiffractionReduction.py

```diff
@@ -30,6 +30,14 @@ def contains_non_ascending_range(strings, delimiter):
     return False
 
 
+def find_minimum_non_zero_y(workspace):
+    y_minimum = min(filter(lambda x: x > 0, workspace.readY(0)))
```

Comment:
It doesn't make a huge difference but I personally find that a list comprehension would be more readable here. `y_minimum = min([x for x in workspace.readY(0) if x>0])`. up to you whether or not you want to here, there execution speed is similar enough to not matter, and the readability is a matter of opinion on this one. just wanted to check you were aware that list comprehension was an option